CONTRIBUTING.md

```diff
@@ -46,19 +50,30 @@ git checkout master
 git pull upstream master
 ```
 
-4. Create a new topic branch:
+4. Install dependencies:
 
 ```sh
-git checkout -b my-topic-branch
+yarn install
 ```
 
-5. Make changes, commit and push to your fork:
+or
+
+````sh
+yarn
+
+5. Create a new topic branch:
+
+```sh
+git checkout -b my-topic-branch
```

Comment:
Is it too soon to start recommending the new syntax?:

```suggestion
git switch -c my-topic-branch
```

Reply:
Are you sure?

Reply:
> $ git switch
git: 'switch' is not a git command. See 'git --help'.

Reply:
That answers my question. 😆 

https://git-scm.com/docs/git-switch

Reply:
@mbrookes I think it is too soon for this syntax. Please have a look at this screenshot taken from git documentation itself.
![git-switch](https://user-images.githubusercontent.com/51196367/71417996-56b2ee80-268e-11ea-8b03-92f9a0120989.png)